PlayCover/View/ChangeGenshinAccountView.swift

```diff
@@ -21,28 +20,34 @@ struct ChangeGenshinAccountView: View {
                 if account != ".DS_Store"{
                     Button(action: {
                         self.folderName = account
-                        restoreUserData(folderName: account)
-                        self.presentationMode.wrappedValue.dismiss()
-                    })
-                    {
+                        self.restoreAlert = true
+                    }, label: {
                         HStack {
                             Image(systemName: "person.fill")
                             Text(account)
                         }.frame(minWidth: 300, alignment: .center)
-                    }.controlSize(.large).buttonStyle(GrowingButton()).font(.title3)
+                    }).controlSize(.large).buttonStyle(.automatic).font(.title3)
+                        .foregroundColor(.accentColor)
                         .frame(width: 300, alignment: .center)
+                        .alert("Really Restore Account?", isPresented: $restoreAlert, actions: {
```

Comment:
I'm not really sure about the message on this alert. Otherwise everything works great. Thank you Candy!